### DISCUSSION TITLE
“The Bumpkin News” - the Sunflower Land newspaper proposal

### DISCUSSION THREAD

**Discussion by drumdrumdrum3337** [sunflower-land/sunflower-land#1367] [Ideas]
Every town, village and yes, even a bumpkins’ settlement, has to have their own newspaper so that all its inhabitants are kept informed, entertained, and feel a sense of togetherness.

**Description**

The Bumpkin News (name is a subject to change) is a periodic newspaper of the Sunflower Land. It could be purchased at a newsstand in the town for a symbolic price (e.g. 0.25 SFL). Every copy of the newspaper is an NFT and is available for purchase until the next edition is out.
The newspaper is owned and created by the dedicated team selected from the discord community, but in coordination with the core team of developers and marketers.
Having an in-game newspaper run by players might be a unique feature of the Sunflower Land game.

**Content of the newspaper**

Sunflower Land news: 

- latest game release explained, 
- curious facts from around the SFL world, 
- tractors racing results (or any other competition results),
- analysis of the price of SFL in crypto markets, as well as prices of resources in the game, etc.

Long(ish) read part: 

- interviews with core devs, community developers, mods, whales, etc., 
- game strategies overviews.

Editorial: a column with thoughts from one of the main people in the game in each edition (i.e. one week column is written by Adam, another by Steve, then by ChickenCX, etc);

Fun corner: 

- meme’s submitted by the community in discord, 
- quizzes, 
- fun gossip, etc.

Overall mood of the newspaper is fun and lightweight, but the aim is to make sure that everyone in the Sunflower Land is on the same page wrt to the current developments in the game. It should also stay flexible and nimble to adapt to the game as it develops.

**Development**

Ideally, everything from the design of a newsstand to the code behind newspaper NFT creation is to be done by the dedicated editorial team selected from the community, and not by the core team. 

The editorial team should consist of:

Chief Bumpkin News Editor
4-5 Bumpkin News Journalists 
1 Bumpkin News Designer
1 Bumpkin News Developer

**Economics**

Selling of the newspaper NFTs will generate some revenue, and if the newspaper is to be published, its distribution needs to be discussed. 

Random example: all SFL revenue is divided into 3 equal parts, where one part goes to the core team, one towards giveaways to readers, and one to the editorial team.

**Important notes**

- Existence of the Bumpkin News would essentially create a precedent of an enterprise functioning within the game and created by the players. This might open up a path to other suggestions of enterprises that would create new game loops that were not initially designed by the core team. 

- The newspaper might eventually include ads. Ads could come either from the players (e.g. artists selling their NFT decorations for Bupkins’ casas, Happy Birthday wishes, etc), or even from the outside of the game (remember discussion in #tokenomics on how to bring cash from the outside into the game?). Distribution of the revenue from ads would have to be discussed as well, but ideally at least the most of it should remain in the game creating some further cash flow.

- If this proposal gets the initial green light, then a much more thorough documentation has to be prepared to include details such as selection process for the editorial team, guidelines for the newspaper content (e.g. specifying what materials are not ok to publish as potentially generating fomo in the community), some graphic designs, etc.

**Comment by adamhannigan** [comment]
Hey @drumdrumdrum3337 👋 

Thanks for the submission. If you haven't seen it already, we covered it in this week's wrap up on Twitch 🤗 

https://www.twitch.tv/videos/1572632158

We love this idea and are happy to move forward. The community should run this newspaper and be incentivised to do so. Essentially your team will get ownership of the contracts + feature, with the idea being you will get more flexibility and control.

From our side, there are a few requirements:

- The "team SFL" can be burnt burnt. We are happy to burn our share as we don't like holding the SFL token (since it means we have to dump it on the community if we want to profit)
- The smart contracts must be open sourced (in your own repo) and the code verified on PolygonScan
- The newspapers themselves should follow the pixel art aesthetic and be PG (no swearing 😅 )

We will be happy to work with your editors to help with developer insights, intel, Q&As etc. ❤️ 

A few small considerations:

- Each newspaper could be a token in an ERC1155 contract (SFT). Each addition would be the same for everyone who bought it
- The newspapers would live in the new community garden. A player will need SFL in their personal wallet (not the farm wallet) to purchase a newspaper)

Do you have in mind a team for the following:

- Smart contracts
- Front-end developers
- Graphic Designer (for newspaper layout)
- Journalists

I would suggest asking around in the developer and artist channels if needed.

Looking forward to seeing this in action 👏 Great proposal

**Reply by drumdrumdrum3337** [reply]
Hi Adam, thanks for taking a look and supporting the free press! I am very excited to see how this will unfold.

Wrt requirements:

- team's SFL - noted on burning. I guess, ideally we'd want to automate the whole process of distribution so that we won't do this manually. Also, I'd suggest we go back to review the distribution after some time, maybe there could be some other area of the game that would use the sfl cash injection.
- smart contracts being open sourced - noted;
- esthetics and design - agree 100%, newspaper should reflect the world it exists in :)

Considerations:
- each copy of the newspaper being an SFT - this is out of my area of expertise, so I'll circle back here when we have a developer in the team in case there are any questions;
- Community garden sounds like exactly the right place for the newsstand (but once the town growth into a city we want a it on the corner!:) 
- Wrt buying with SFL from the personal wallet, this is again discussion for a bit later, when we have a developer on board. But, one note is that we would want to build it in a way that would later allow us to do some activities with dedicated readers, e.g. perform snapshots to see who holds all the editions, or to be able to do some sort of a lottery thing where each newspaper would be a lottery ticket (every civilized society has a lottery, right?:). But, again, this is a discussion for a tad bit later.

Team: I don't have it yet, I'll start asking around while also preparing some basic foundational docs (would be happy to share them with you). I'll check with the person you mentioned during the stream as well.

Thank you!
Mykhaylo.

**Comment by Q4FOIL** [comment]
**The Sunflower Times**

**Comment by drumdrumdrum3337** [comment]
Quick update on the progress:

- JcEii is working on the first newspaper mockup;
- Tiffany will bring up some technical aspects with Adam/team after Frogs are released;
- I am in the middle of completing foundational documents for the newspaper.

Once these are completed, we'll be sharing all the materials with the core team for comments. Once we are aligned with the core team, invites to apply for the roles in the Bumpkin News team will be posted in the discord.
